test/Project.toml

```diff
@@ -0,0 +1,23 @@
+[deps]
```

Comment:
We need compat entries here, otherwise it's going to be difficult to ensure that we're running in the correct environment

Reply:
Okay

Reply:
done

Reply:
done